src/linker/Linker.Steps/OutputStep.cs

```diff
@@ -271,42 +271,27 @@ FileInfo GetOriginalAssemblyFileInfo (AssemblyDefinition assembly)
 
 		protected virtual void CopyAssembly (AssemblyDefinition assembly, string directory)
 		{
-			// Special case.  When an assembly has embedded pdbs, link symbols is not enabled, and the assembly's action is copy,
-			// we want to match the behavior of assemblies with the other symbol types and end up with an assembly that does not have symbols.
-			// In order to do that, we can't simply copy files.  We need to write the assembly without symbols
-			if (assembly.MainModule.HasSymbols && !Context.LinkSymbols && assembly.MainModule.SymbolReader is EmbeddedPortablePdbReader) {
-				WriteAssembly (assembly, directory, new WriterParameters ());
-				return;
-			}
-
 			FileInfo fi = GetOriginalAssemblyFileInfo (assembly);
 			string target = Path.GetFullPath (Path.Combine (directory, fi.Name));
 			string source = fi.FullName;
+
+			if (assembly.MainModule.HasSymbols && !Context.LinkSymbols && assembly.MainModule.SymbolReader is EmbeddedPortablePdbReader)
+				Context.LogWarning ("Debug symbols cannot be modified in a copied assembly.", 2104, new MessageOrigin (source));
```

Comment:
Why is it a warning?

Reply:
We used to allow symbols to be removed from copy assemblies; this would inform users that were doing this that the scenario is no longer supported. Although I think it's a fair point that it doesn't make sense to ask for removal in something tagged with "copy" -- I'll remove warning here.